lib/services/bootstrap/internalBootstrap.js

```diff
@@ -26,14 +26,15 @@ const
   crypto = require('crypto'),
   errorsManager = require('../../config/error-codes/throw');
 
+// @todo rename to InternalIndexBootstrap
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)